README.md

```diff
@@ -6,8 +6,8 @@
 * **Compatible Version**: 7.6.0 and later
 * [Release Notes](./release_notes.md)
 
->[!NOTE]
->This is a preview version of **FortiAI** designed to demonstrate the potential of Generative AI in FortiSOAR. Preview releases are a beta release. This means that release is intended to get feedback and might not be best suited for production level deployments. Feature availability and support for preview releases will continue to improve as the solution/feature matures.
+> [!NOTE]
+> This is a preview version of **FortiAI** designed to demonstrate the potential of Generative AI in FortiSOAR. Preview releases are a beta release. This means that release is intended to get feedback and might not be best suited for production level deployments. Feature availability and support for preview releases will continue to improve as the solution/feature matures.
 
 # Overview
```

Comment:
Overview should also now talk about connector generation

Reply:
In overview, like we mentioned about "Filtering Feature Overview" we should now also add about advance PB generation and connector generation before "Filtering Feature Overview"